repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -66,7 +66,18 @@ object ProjectUpdateRequest {
 
 final case class GitLabVersion(name: PipelineVersion, message: String, target: String, commit: Commit)
 object GitLabVersion {
-  implicit val gitlabVersionFormat: OFormat[GitLabVersion] = Json.format[GitLabVersion]
+
+  implicit object gitlabVersionFormat extends Format[GitLabVersion] {
+
+    override def writes(o: GitLabVersion): JsValue = JsString(o.toString)
+
+    override def reads(json: JsValue): JsResult[GitLabVersion] = for {
+      name <- (json \ "name").validate[PipelineVersion]
+      message <- (json \ "message").validate[String]
+      target <- (json \ "target").validate[String]
+      commit <- (json \ "commit").validate[Commit]
+    } yield GitLabVersion(name, message, target, commit)
+  }
```

Comment:
Why did you change this formatter?
Also, have you tried to serialize and then deserialize an instance of `GitLabVersion` with this formatter? It looks like it won't work.

Reply:
Changed back. Unfortunately, I forgot why. There is no proper reason to change it.